app/controllers/exception_hunter/errors_controller.rb

```diff
@@ -2,14 +2,21 @@
 
 module ExceptionHunter
   class ErrorsController < ApplicationController
+    include Pagy::Backend
```

Comment:
We need to explicitly require pagy in order for this to work. That can be done on the engine file